app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx

```diff
@@ -333,36 +331,22 @@ export class SimpleWidget extends Component {
   };
 
   refreshWidget = () => {
-    this.props.tracking.trackEvent(DASHBOARD_PAGE_EVENTS.REFRESH_WIDGET);
+    this.props.tracking.trackEvent(WIDGETS_EVENTS.CLICK_ON_REFRESH_WIDGET_ICON);
     this.fetchWidget();
   };
 
   showDeleteWidgetModal = () => {
-    const { tracking, isAnalyticsEnabled, onDelete, baseEventParameters } = this.props;
+    const { tracking, isAnalyticsEnabled, onDelete } = this.props;
 
-    tracking.trackEvent(DASHBOARD_PAGE_EVENTS.REMOVE_WIDGET);
+    tracking.trackEvent(WIDGETS_EVENTS.CLICK_ON_DELETE_WIDGET_ICON);
     const onConfirm = () => {
       const { widgetId, activeDashboardId, widgetType } = this.props;
-      const {
-        widget: { id },
-      } = this.state;
 
       onDelete(widgetId);
       if (isAnalyticsEnabled) {
-        provideEcGA({
-          eventName: 'remove_from_cart',
-          baseEventParameters,
-          additionalParameters: {
-            item_list_name: activeDashboardId,
-            items: [
-              getEcWidget({
-                itemId: id,
-                itemName: widgetTypesMessages[widgetType].defaultMessage,
-                itemListName: activeDashboardId,
-              }),
-            ],
-          },
-        });
```

Comment:
I'm not sure it should be deleted as it is was done recently within the EPMRPP-87528.
Please discuss with Web Analyst.